### PR TITLE
Issues #SB-00 feat:changes for user api's

### PR DESCRIPTION
**Pull request by manzarul**

### CODE REVIEW

**Comment by lakhanmandloi** [comment]
Hello @manzarul,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://407.qa.docs.sunbird.org/master/ .<br/> **Note:** Only _master_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://407.qa.docs.sunbird.org/master/apis/userapi// <br />  <br/> **Modified sections are -** <br />

**Comment by lakhanmandloi** [comment]
Hello @manzarul,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://407.qa.docs.sunbird.org/master/ .<br/> **Note:** Only _master_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://407.qa.docs.sunbird.org/master/apis/userapi// <br />  <br/> **Modified sections are -** <br />

**Comment by lakhanmandloi** [comment]
Hello @manzarul,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://407.qa.docs.sunbird.org/master/ .<br/> **Note:** Only _master_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://407.qa.docs.sunbird.org/master/apis/userapi// <br />  <br/> **Modified sections are -** <br />

File: apis/userapi/userapi.yaml [hunk 1]
```diff
@@ -79,12 +78,39 @@ paths:
...
-        AllowDynamicQueryParameters: false
-        AllowDynamicFormParameters: false
-        IsMultiContentStreaming: false
+      x-unitTests: 
+      - example1:
+          description: 'example for user create workflow'
+          body: >-
+            {
+                "request": {
+                    "firstName": "combination of firstname and lastname",
```
Comment (NupurSancheti) [inline apis/userapi/userapi.yaml]: first name and last name???

Comment (NupurSancheti) [reply apis/userapi/userapi.yaml]: separated by space

File: apis/userapi/userapi.yaml [hunk 2]
```diff
@@ -79,12 +78,39 @@ paths:
...
+                    "email":"valid email id",
+                    "emailVerified":true,
+                    "password":"user password",
+                    "channel":"channel value of user rootOrg",
```
Comment (NupurSancheti) [inline apis/userapi/userapi.yaml]: Manzarul: same value when the org is created

**Comment by Sunitashelar** [comment]
closing the PR without merging it as there are unresolved comments

**Pull request closed without merging**